Resources/Prototypes/Catalog/VendingMachines/Inventories/janidrobe.yml

```diff
@@ -4,6 +4,7 @@
     ClothingUniformJumpsuitJanitor: 2
     ClothingUniformJumpskirtJanitor: 2
     ClothingHandsGlovesJanitor: 2
+    ClothingShoesGaloshes: 2
```

Comment:
Could you add a comment here stating this is a CD change?

Reply:
Added the comment, let me know if I need to edit it

Reply:
Yep, that works.